jsonrpc/eth_endpoint.go

```diff
@@ -490,6 +490,10 @@ func (e *Eth) Call(arg *txnArgs, filter BlockNumberOrHash, apiOverride *stateOve
 
 // EstimateGas estimates the gas needed to execute a transaction
 func (e *Eth) EstimateGas(arg *txnArgs, rawNum *BlockNumber) (interface{}, error) {
+	if arg == nil {
+		return nil, errors.New("missing value for required argument 0")
```

Comment:
Can we move this to `DecodeTxn`? It is used in other rpc endpoints, so we can tackle the same error on a single place.

Reply:
I agree with Goran. Because right now we are fixing problem only in one place